### PR TITLE
HUE-9176 [core] Introduce Vue dependency

### PR DESCRIPTION
**Pull request by enricoberti**
Complete with vue-loader that compiles .vue single-file components.
A sample component is included, as is a tiny bit of documentation in the hue.js entry point

### CODE REVIEW

**Comment by romainr** [comment]
Nice!

**Comment by enricoberti** [comment]
Updated with a working sample component on the login page

File: desktop/core/src/desktop/js/hue.js [hunk 1]
```diff
@@ -74,6 +74,24 @@ import HueFileEntry from 'doc/hueFileEntry';
 import HueDocument from 'doc/hueDocument';
 import { REFRESH_CONFIG_EVENT } from 'utils/hueConfig';
 
+// import Vue from 'vue';
```
Comment (romainr) [inline desktop/core/src/desktop/js/hue.js:77]: Remove?

**Comment by romainr** [comment]
Looks good!

i18n might be an issue but maybe find the official way to tag strings or do later?

**Comment by romainr** [comment]
Will update the CI image so that the build passes

**Comment by romainr** [comment]
Ha, might need a rebase now?

**Comment by enricoberti** [comment]
@romainr solved merge conflicts!

**Pull request merged**